apps/meteor/app/api/server/helpers/parseJsonQuery.ts

```diff
@@ -56,7 +52,7 @@ export async function parseJsonQuery(
 
 	let fields: Record<string, 0 | 1> | undefined;
 	if (params.fields) {
-		warnFields('attribute fields is deprecated');
+		apiDeprecationLogger.parameter(this.request.route, 'fields', '7.0.0', this.response);
```

Comment:
same practical effect, but the idea is not to release a 7.0.0 with deprecations for it, this will force us to at least revisit one by one

version + 1 is very convenient and easy to leave forever